src/amo/components/RatingsByStar/index.js

```diff
@@ -86,85 +90,104 @@ export class RatingsByStarBase extends React.Component<InternalProps> {
     );
   }
 
+  handleRouting(star: string) {
+    const { addon, location, history, clientApp, siteLang } = this.props;
+    history.push({
+      pathname: `/${siteLang}/${clientApp}/addon/${addon.slug}/reviews/`,
+      search: `?score=${star}&src=${location.query.src}`,
+    });
+  }
+
   render() {
-    const { addon, errorHandler, i18n, groupedRatings, location } = this.props;
+    const { addon, errorHandler, i18n, groupedRatings } = this.props;
     const loading = (!addon || !groupedRatings) && !errorHandler.hasError();
-
-    const linkTitles = {
-      /* eslint-disable quote-props */
-      '5': i18n.gettext('Read all five-star reviews'),
-      '4': i18n.gettext('Read all four-star reviews'),
-      '3': i18n.gettext('Read all three-star reviews'),
-      '2': i18n.gettext('Read all two-star reviews'),
-      '1': i18n.gettext('Read all one-star reviews'),
-      /* eslint-enable quote-props */
+    const self = this;
+
+    const getLinkTitle = (star, count = '') => {
+      switch (star) {
+        case '5':
+          return i18n.gettext(`Read all ${count} five-star reviews`);
+        case '4':
+          return i18n.gettext(`Read all ${count} four-star reviews`);
+        case '3':
+          return i18n.gettext(`Read all ${count} three-star reviews`);
+        case '2':
+          return i18n.gettext(`Read all ${count} two-star reviews`);
+        case '1':
+          return i18n.gettext(`Read all ${count} one-star reviews`);
+        default:
+          return '';
+      }
     };
 
     return (
       <div className="RatingsByStar">
         {errorHandler.renderErrorIfPresent()}
-        <div className="RatingsByStar-graph">
-          {['5', '4', '3', '2', '1'].map((star) => {
-            let starCount;
-            let starCountNode;
-
-            function createLink(text) {
-              invariant(addon, 'addon was unexpectedly empty');
+        <table className="RatingsByStar-graph">
+          <tbody>
+            {['5', '4', '3', '2', '1'].map((star) => {
+              let starCount;
+              let starCountNode;
+
+              function createLink(body) {
+                invariant(addon, 'addon was unexpectedly empty');
+
+                return (
+                  <tr
+                    title={getLinkTitle(star, starCount) || ''}
```

Comment:
`getLinkTitle` returns `''` as the `default`, so why do you need the `|| ''` here?

Reply:
Yeah, I added this as extra check type but yes can be removed. I will updated that.